src/cluster.h

```diff
@@ -116,4 +116,7 @@ int isValidAuxString(char *s, unsigned int length);
 void migrateCommand(client *c);
 void clusterCommand(client *c);
 ConnectionType *connTypeOfCluster(void);
+
+int detectAndUpdateCachedNodeHealth(void);
```

Comment:
This is not a function with a shared implementation, like the header says, so it should go up in the previous section.